### DISCUSSION TITLE
what is the meaning of TipMass?

### DISCUSSION THREAD

**Discussion by tatesnow** [OpenFAST/openfast#953] [Q&A] [answered]
I am reading the ElastoDyn users guide and Theory Manual, and I don't know the meaning of "TipMass" and "Tip-brake" in the input file of  ElastoDyn.

**Comment by jjonkman** [comment, marked as answer]
Dear @tatesnow,

The TipMass refers to a lumped point mass (without rotational inertia) located at the tip node of the blade that can be specified separately from the blade distributed mass.  Some wind turbine blades include such a tip mass.

The tip-brake feature was included in FAST v7, but is not available in OpenFAST.  This referred to an aerodynamic brake installed at the tip of the blade that could be deployed to slow down the rotor.  Some wind turbine blades in the past (that were much smaller than today's utility scale wind turbines) included such devices.

Best regards,

**Comment by tatesnow** [comment]
Thank you very much!
